Tests/BrowserServicesKitTests/ContentBlocker/WebViewTestHelper.swift

```diff
@@ -85,6 +86,10 @@ final class MockSurrogatesUserScriptDelegate: NSObject, SurrogatesUserScriptDele
         return shouldProcessTrackers
     }
 
+    func surrogatesUserScriptShouldProcessCTLTrackers(_ script: SurrogatesUserScript) -> Bool {
+        return shouldProcessCTLTrackers
```

Comment:
you can skip "return" keyword and make it 1-liner ... -> Bool { shouldProcessCTLTrackers }

Reply:
Agreed, though I'm copying the pattern used elsewhere in the file (ie. immediately above in `surrogatesUserScriptShouldProcessTrackers()` and elsewhere).

Happy to change it but then we're introducing an inconsistency, unless you want me to change the others?

Reply:
The inconsistency is due to this code being written before skipping the "return" keyword was a common practice in Swift for one-liners. I think it makes sense to leave old code as is, but adopt the new patterns in new code or where possible. Otherwise, we would need to find and replace all instances in our app, or we would never adopt the new patterns.